### ISSUE TITLE
Option for UnusedVariableSniff with usage of list()

### ISSUE THREAD

**Issue by mr-feek** [slevomat/coding-standard#561]
Here is a code example that would ideally pass with `UnusedVariableSniff` applied, given a new option similar to `ignoreUnusedValuesWhenOnlyKeysAreUsedInForeach`.

```
<?php

list($foo, $bar) = ['foo', 'bar'];

echo $bar;

```

Expected: no errors
Actual: ERROR | Unused variable $foo.

**Comment by kukulich** [comment]
@mr-feek 

You can use

```php
list(, $bar)
```

**Comment by mr-feek** [comment]
hah! Today I learned.
